packages/graph-engine/src/nodes/color/matchAlpha.ts

```diff
@@ -0,0 +1,155 @@
+import { Black, Gray, White, toColor, toColorObject } from './lib/utils.js';
+import {
+	BooleanSchema,
+	ColorSchema,
+	NumberSchema
+} from '../../schemas/index.js';
+import { INodeDefinition } from '../../index.js';
+import { Node } from '../../programmatic/node.js';
+import { setToPrecision } from '@/utils/precision.js';
+import Color from 'colorjs.io';
+
+export default class NodeDefinition extends Node {
+	static title = 'Match Alpha';
+	static type = 'studio.tokens.color.matchAlpha';
+	static description =
+		'Finds the alpha value that, when used to blend the foreground and background colors, will result in the reference color.';
+	constructor(props: INodeDefinition) {
```

Comment:
You can use the declare syntax to make this more obvious for typescript users
eg
```
	declare inputs: ToInput<{
		top: number;
		right: number;
		bottom: number;
		left: number;
	}>;
	declare outputs: ToOutput<{
		value: string;
	}>;
	```

Reply:
How to put default values in this format?